R/aggregate_funs.R

```diff
@@ -44,16 +44,10 @@
 #' aggzones <- SpatialPolygonsDataFrame(aggzones, data.frame(region = c(1:6)), match.ID = FALSE)
 #' sp::proj4string(aggzones) = sp::proj4string(zones)
 #' od_ag = od_aggregate(flow, zones, aggzones)
-#' od_sp = od2line(od_agg, zones)
-#' zones@data = cbind(1:nrow(zones), zones@data)
-#' od_sp_agg = od2line(od, zones, aggzones)
-#' # plot results
-#' plot(aggzones, lwd = 5)
-#' plot(zones, border = "red", add = TRUE)
-#' plot(od_sp, add = TRUE, col = "yellow")
-#' lwd = od_sp_agg$All / 50
-#' plot(od_sp_agg, lwd = lwd, add = TRUE)
-#'
+#' od_ag = na.omit(od_ag)
```

Comment:
Heads-up @richardellison I have to include this from preventing the next line from generating an error, because the Spatial version seems to generate many NAs in the id fields `o_region` and `d_region`. Any ideas? I think this is why the results, although reasonable, seem distorted now:

Reply:
![image](https://user-images.githubusercontent.com/1825120/30454512-7e4aca5a-9994-11e7-89ef-e7eda7acdfb9.png)